Configurations/10-main.conf

```diff
@@ -29,20 +32,23 @@ sub vc_win32_info {
         my $ver=`nasm -v 2>NUL`;
         my $vew=`nasmw -v 2>NUL`;
         if ($ver ne "" || $vew ne "") {
-            $vc_win32_info = { as        => $ver ge $vew ? "nasm" : "nasmw",
+            $vc_win32_info = { AS        => $ver ge $vew ? "nasm" : "nasmw",
+                               ASFLAGS   => "",
                                asflags   => "-f win32",
-                               asoutflag => "-o",
+                               ASOUTFLAG => "-o",
```

Comment:
Same here, i.e. space after `-o`. Just in case, there is space in 1.0.2.

Reply:
Will fix